data/brands/leisure/fitness_centre.json

```
@@ -13,6 +13,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "EASYFITNESS",
+      "locationSet": {"include": ["at","de","es"]},
+      "tags": {
+        "brand": "EASYFITNESS",
+        "brand:wikidata": "Q106166703",
+        "leisure": "fitness_centre",
+        "name": "EASYFITNESS"
```

```suggestion
      "displayName": "Easyfitness",
      "locationSet": {"include": ["at","de","es"]},
      "tags": {
        "brand": "Easyfitness",
        "brand:wikidata": "Q106166703",
        "leisure": "fitness_centre",
        "name": "Easyfitness"
```
Title case is [generally preferred](https://wiki.openstreetmap.org/wiki/Names#Uppercase_characters) for names.